uktrade/data-hub-frontend
# Add `Other countries the investor is considering` field to Large Capital Profile

## Description of change

Add `Other countries the investor is considering` field to location section of large capital profiles.

A branch off of my previous PR as it isn't yet merged and this work is dependent on it - https://github.com/uktrade/data-hub-frontend/pull/2103 . So it is just the most recent commit, `95a9fed`, that is relevant to this PR. I will rebase everything once the previous PR is approved and merged. 

## Test instructions

After clicking edit on the location section you should see an `Add another` select field displaying active country metadata. You should be able to select multiple countries and save them to the profile. The profile should then display the countries saved. 
 
## Screenshots
### Before

![Screenshot 2019-09-02 at 17 18 01](https://user-images.githubusercontent.com/42253716/64126518-f1b54f00-cda5-11e9-87e9-5919dd40ca73.png)
![Screenshot 2019-09-02 at 17 20 02](https://user-images.githubusercontent.com/42253716/64126519-f24de580-cda5-11e9-8a78-1404f81ab556.png)

### After 

![Screenshot 2019-09-04 at 18 37 12](https://user-images.githubusercontent.com/42253716/64277784-1c88da00-cf43-11e9-8395-f5c20cd34ab8.png)
![Screenshot 2019-09-04 at 18 37 32](https://user-images.githubusercontent.com/42253716/64277785-1c88da00-cf43-11e9-8665-72eb53e9e6f6.png)

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied. 
https://github.com/uktrade/data-hub-frontend/blob/develop/docs/Code%20review%20guidelines.md"
